tests/negative/gold/illegal_system_call_test.gold

```diff
@@ -0,0 +1,9 @@
+As check failed: null is not a ServiceResource.
```

Comment:
Before this change, this test would have generated a complete nonsense stack trace.